docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md

```diff
@@ -281,137 +146,318 @@ export const httpLoaderFactory = (httpClient: HttpClient) => {
 export class AuthConfigModule {}
 ```
 
-### Return type of `checkAuth()` method is now an object `LoginResponse`
+## Configuration
 
-In the previous version the `checkAuth` method only returned if you are authenticated or not. New the method returns the `LoginResponse` object which provides
+### `authWellknownEndpoint` renamed to `authWellknownEndpointUrl`
 
-```typescript
-export interface LoginResponse {
-  isAuthenticated: boolean;
-  userData?: any;
-  accessToken?: string;
-  errorMessage?: string;
+Old:
+
+```ts
+export interface OpenIdConfiguration {
+  authWellknownEndpoint?: string;
+  // ...
 }
 ```
 
-#### Old
+New:
 
-```typescript
-this.oidcSecurityService.checkAuth().subscribe((isAuthenticated) => {
-  console.log('app authenticated', isAuthenticated);
-});
+```ts
+export interface OpenIdConfiguration {
+  authWellknownEndpointUrl?: string;
+  // ...
+}
 ```
 
-#### New
+### AuthWellKnownEndpoints are now part of the config
 
-```typescript
-this.oidcSecurityService.checkAuth().subscribe(({ isAuthenticated }) => {
-  console.log('app authenticated', isAuthenticated);
-});
+In the previous version the `AuthWellKnownEndpoints` have been a special parameter which you could pass. The type is the same but the property is now part of the configuration. The parameter to pass is in the `withConfig` method was removed.

```

Comment:
For clarity, I would rephrase this to be:

In the previous version, `AuthWellKnownEndpoints` was a separate parameter you could provide alongside the config in `withConfig` to configure your STS Server's Well-Known Endpoints. This parameter still exists, and it is of the same type, but it's now a parameter on the config object rather than being a separate parameter on `withConfig`